contracts/axelar-interchain-token-service/src/contract.rs

```diff
@@ -0,0 +1,100 @@
+use axelar_soroban_std::ensure;
+use soroban_sdk::{contract, contractimpl, Address, Env, String,};
+
+use crate::error::InterchainTokenServiceError;
+use crate::event;
+use crate::storage_types::{DataKey, TrustedItsAddress};
+
+#[contract]
+pub struct InterchainTokenService;
+
+#[contractimpl]
+impl InterchainTokenService {
+    pub fn initialize(
```

Comment:
```suggestion
    pub fn initialize_its(
```
naming collisions aren't allowed when one contract imports another in soroban